cmd/benchmark.go

```diff
@@ -85,6 +85,9 @@ func setupBenchmarkCommand() *cobraext.Command {
 	systemCmd := getSystemCommand()
 	cmd.AddCommand(systemCmd)
 
+	validateCmd := getValidateCommand()
+	cmd.AddCommand(validateCmd)
```

Comment:
Instead of adding this as a new subcommand, I think we could add this as part of one of the subcommands we have now for validation. This would have the benefit of being executed by CI.

I think a good place could be in `elastic-package test static`, there we are only checking the sample events, but checking the events generated by `stream` could also fit there.

Other place could be `elastic-package check`, but this is more concerned about the structure of the package more than its behavior.

Reply:
It makes sense to me to test as much as possible via a central command - I added this to `test static`, but IMHO there is value in having a dedicated command as well to be able to iterate on only the benchmark config.

What do you think about the most recent version that wires up the same logic in both places?